front/src/modules/ui/view-bar/components/MultipleFiltersDropdownButton.tsx

```diff
@@ -8,17 +10,35 @@ import { ViewBarDropdownButton } from './ViewBarDropdownButton';
 
 type MultipleFiltersDropdownButtonProps = {
   hotkeyScope: HotkeyScope;
+  isInViewBar?: boolean;
 };
 
+const StyledDropdownContainer = styled.div<{ isInViewBar?: boolean }>`
+  ${({ isInViewBar }) =>
+    isInViewBar &&
+    `
+      left: 0px;
+      position: absolute;
+      top: 32px;
+      z-index: 1;
+      background-color: white;
+    `}
+`;
+
 export const MultipleFiltersDropdownButton = ({
   hotkeyScope,
+  isInViewBar,
 }: MultipleFiltersDropdownButtonProps) => {
   return (
-    <ViewBarDropdownButton
-      dropdownId={FilterDropdownId}
-      buttonComponent={<MultipleFiltersButton />}
-      dropdownComponents={<MultipleFiltersDropdownContent />}
-      dropdownHotkeyScope={hotkeyScope}
-    />
+    <StyledDropdownContainer isInViewBar={isInViewBar}>
+      <ViewBarDropdownButton
+        dropdownId={isInViewBar ? hotkeyScope.scope : FilterDropdownId}
```

Comment:
Not sure if using a hotkeyScope as a dropdownId is relevant, could we create another dropdown id ? Do we need a dynamic one ? We could add a new props dropdownId in `MultipleFiltersDropdownButton` as a default way of doing this.

Reply:
Let us look into this @lucasbordeau